### PR TITLE
fix(tooltip): tooltip will be cut off at the limits of the innermost element with hidden overflow

### PR DESCRIPTION
**Pull request by boiarchukk**
The popper now initially visible in case when parent boundary element has overflow: hidden
## Description

Due to the Popper [limitation](https://github.com/popperjs/popper-core/issues/112) it will be cut off at the limits of the innermost element with overflow: hidden; To resolve this issue, Popper.js instance need to be setting up in the md-theme, because this element guarantee will not contain overflow property with hidden value.
## Motivation and Context

Before this fix, Popper cut off when target element is inside element with overflow: hidden
## How Has This Been Tested?

Manually, UI tests

## Screenshots:
**Before** (If applicable):

Screenshot at Nov 17 16-19-59

**After**:

Снимок экрана 2020-11-17 в 17 19 41

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by elenagiere** [comment]
This seems to have broken all tooltips. Even a simple tooltip in sandbox renders at the top left with no styling and flickers like an infinite loop has been hit. I can't screenshot it because it flickers so much.

**Comment by futurethang** [comment]
@boiarchukk This isn't appearing to work well at the moment. I'm not sure about having so much tooltip specific logic all living in the Theme component either, it's an odd overlapping of interests. One buggy aspect id that the @mouseover fires continuously and makes things flicker, aside from tooltips looking broken all around.
I know this is incomplete, but it seems like it's jumping through a lot of hoops to arrive at a solution that's still out of reach.

**Comment by boiarchukk** [comment]
@futurethang @elenagiere Check again please. Previously it was draft changes and they not reflected actually progress state

File: web-components/src/components/tooltip/Tooltip.ts [hunk 1]
```diff
@@ -44,120 +45,83 @@ export namespace Tooltip {
 @customElement("md-tooltip")
 export class Tooltip extends FocusMixin(LitElement) {
   @property({ type: String }) message = "";
-  @property({ type: String }) placement: Tooltip.Placement = "auto";
-  @property({ type: Boolean }) disabled = false;
```
Comment (elenagiere) [inline web-components/src/components/tooltip/Tooltip.ts:48]: @boiarchukk Why are you removing disabled? We really need this. Now all tooltips are enabled always.

File: web-components/src/components/tooltip/Tooltip.test.ts [hunk 2]
```diff
@@ -34,52 +34,52 @@ describe("Tooltip", () => {
     expect(true).toBeTruthy();
   });
 
-  test("should show/hide Tooltip Component", async () => {
-    const element: Tooltip = await fixtureFactory("Test text", false);
-    element.showTooltip();
-    await elementUpdated(element);
+  // test("should show/hide Tooltip Component", async () => {
```
Comment (futurethang) [inline web-components/src/components/tooltip/Tooltip.test.ts:37]: So now do we need these tests to remain or can we clear them out and move coverage to the Theme component?

**Review by futurethang — APPROVED** [review]

**Comment by elenagiere** [comment]
@boiarchukk Looks great! Great job figuring out this difficult bug. I have tested in all throughout Agentx and works better than ever!
I did notice, that sometimes the slotted content isn't styled perfectly:
Screen Shot 2020-11-17 at 9 07 25 AM

This may just be changes we need to make in the mfe repos.

**Review by elenagiere — APPROVED** [review]

**Pull request merged**